### PR TITLE
Some refactoring of the docs/ content

### PR DESCRIPTION
**Pull request by timvink**

### CODE REVIEW

**Comment by timvink** [comment]
Some ❤️  for the docs. Some broken links and TODO's removed, restructured some content.

I still feel there's a bit of duplication, some things can be condensed etc. Needs some more work still.

File: docs/howto/save_buckets_to_file.ipynb [hunk 1]
```diff
@@ -4,15 +4,26 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Save buckets to files\n",
+    "# Saving bucket information to a file\n",
     "\n",
-    "After the bucketing process has been completed, it's useful to save the buckets to a file.<br>\n",
-    "`skorecard` supports saving bucketers to yaml and json files."
+    "If you have a specific set of bucketing boundaries you are satisfied with, it's useful to save them to a file.\n",
+    "You might want to save the bucketing information a configuration files along with your code.\n",
```
Comment (orchardbirds) [inline docs/howto/save_buckets_to_file.ipynb:10]: 'and' configuration files

**Review by orchardbirds — COMMENTED** [review]
smallest suggestion haha, the rest looks great!

File: docs/howto/save_buckets_to_file.ipynb (hunk 1 above)
Comment (timvink) [reply docs/howto/save_buckets_to_file.ipynb:10]: 'as' configuration files actually. Typo fixed thnx

**Pull request merged**